src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -507,18 +508,20 @@
                     </Button>
                 </StackPanel>
             </Border>
-            <Border x:Name="TracePointer"
-                    Width="16"
-                    Height="18"
-                    Margin="48,50,0,0"
-                    HorizontalAlignment="Left"
-                    VerticalAlignment="Top"
-                    Visibility="Collapsed">
-                <FontIcon Foreground="Red"
-                          FontFamily="{StaticResource CalculatorFontFamily}"
-                          FontSize="18"
-                          Glyph="&#xE3B3;"/>
-            </Border>
+            <Grid x:Name="TracePointer"
+                  HorizontalAlignment="Left"
+                  VerticalAlignment="Top"
+                  Visibility="Collapsed">
+                <Border x:Name="CursorShadow"/>
+                <Path x:Name="CursorPath"
+                      Width="18"
+                      Height="18"
+                      Fill="White"
```

Comment:
Fill and Stroke should support high contrast (Suggestion: Fill = SystemColorHighlightTextColor and Stroke = SystemColorWindowText to contrast with the background)

Reply:
(Windows doesn't change the color of the cursor when you switch from normal mode to high contrast mode, but I suppose users also change the size and the color of their cursors to better see it)

Reply:
fair.  will update the colors.